cabal-install/Distribution/Client/CmdClean.hs

```diff
@@ -22,46 +22,52 @@ import Distribution.Simple.Utils
 import Distribution.Verbosity
     ( Verbosity, normal )
 
+import Control.Monad
+    ( mapM_ )
 import Control.Exception
     ( throwIO )
 import System.Directory
-    ( removeDirectoryRecursive, doesDirectoryExist )
+    ( removeDirectoryRecursive, removeFile
+    , doesDirectoryExist, getDirectoryContents )
+import System.FilePath
+    ( (</>) )
 
 data CleanFlags = CleanFlags
-    { cleanSaveConfig :: Flag Bool
-    , cleanVerbosity :: Flag Verbosity
-    , cleanDistDir :: Flag FilePath
+    { cleanSaveConfig  :: Flag Bool
+    , cleanVerbosity   :: Flag Verbosity
+    , cleanDistDir     :: Flag FilePath
```

Comment:
Style curiosity question: why is this formatting so common? It results in so much whitespace noise in diffs because whenever you remove, rename, or add a longer entry you need to re-indent everything else. You also end up with inconsistency when one of the entries is really long and needs to hang on the next line (or results in the other entries being separated by a lot of whitespace).

Reply:
It's subjective of course, but having things nicely aligned helps you see the structure rightaway; if it isn't aligned it's more diff friendly but otoh it adds cognitive overhead for the human programmer to see the structure. Also, it's not that bad; you can come up with heuristics for realigning which reduce the diff-noise and deal with overlong entries...

Reply:
GitHub diff viewer allows ignoring whitespace noise in diffs (`?w=1` IIRC). I also usually separate formatting commits from meaningful ones.